lfs/gitscanner.go

```diff
@@ -95,6 +95,25 @@ func (s *GitScanner) ScanRangeToRemote(left, right string, cb GitScannerFoundPoi
 	return scanLeftRightToChan(s, callback, left, right, s.cfg.OSEnv(), s.opts(ScanRangeToRemoteMode))
 }
 
+// ScanMultiRangeToRemote scans through all commits starting at the left ref but
+// not including the right ref (if given)that the given remote does not have.
```

Comment:
Nit: might add a space after `(if given)`, but I notice the preceding comment is the same, so maybe not?

Reply:
Yeah, I copied and pasted, so will fix.  Thanks for noticing.